docs/ADOPTING_DARIA.md

```diff
@@ -0,0 +1,43 @@
+# Adopting DARIA
+
+If you are a fund considering adopting DARIA as your case management system of record, this may answer common questions about how things work. We also have a sandbox/training app where no actual patient data lives for experimenting. [FILL IN ANY NECESSARY DETAIL]
```

Comment:
```suggestion
If you are a fund considering adopting DARIA as your case management system of record, this may answer common questions about how things work. We also have a sandbox/training app where no actual patient data lives for experimenting. You can check out the sandbox [here](https://sandbox.dcabortionfund.org) with username `test@test.com`, password `P4ssword`. Be sure to check out the 'Admin Tools' section to understand how DARIA can be customized to your organization.

If after reading below, you feel that DARIA's a good fit for your organization, just reach out to us on the Code4DC Slack -- [sign up here](https://codefordc.org/resources/slack.html) -- in the #dcaf_case_management channel.
```

Reply:
^^ my crack at it. Assuming this is how you want people to reach out until things are 100% sorted out.